src/concrete/ml/sklearn/xgb.py

```diff
@@ -32,7 +33,7 @@ def __init__(
         max_depth: Optional[int] = 3,
         learning_rate: Optional[float] = None,
         n_estimators: Optional[int] = 20,
-        objective: Optional[str] = "binary:logistic",
```

Comment:
we should check that this does not change the output and thus the way we do post-processing though 🤔

Reply:
let's see what the tests say

Reply:
Actually their was a bug in how I grabbed the default param. I added back the objective. It was correct.